docs/database.rst

```diff
@@ -0,0 +1,40 @@
+rotki Database
+##############
+
+rotki uses two different sqlite databases, one with information about assets (global.db) and one with user information (rotkehlchen.db).
+The latter is encrypted using an extension called `SQLCipher <https://github.com/sqlcipher/sqlcipher>`__
+that provides transparent 256-bit AES full database encryption.
+
+Database Location
+=================
+
+The default database locations are:
+
+- **Linux**: ``~/.share/local/rotki/data``
+- **OSX**: ``??``
+- **Windows**: ``??``
+
+Exploring the database
+**********************
+
+To open the user database you can use `sqlitebrowser <https://sqlitebrowser.org/>`__.
+It supports sqlcipher and will ask for the password used to decrypt the database.
+
+If you prefer the command line instead, you can use the ``sqlcipher`` cli tool. Note: using the ``sqlite3`` cli will not work!
+
+Note to Debian and Ubuntu users: we are using SQLCipher encryption v4, therefore a recent version of sqlcipher is required. Unfortunately
+the version available in the distribution repositories are too old and won't let you open the database using ``sqlitebrowser`` nor ``sqlcipher``.
+On top of that, the ``sqlitebrowser`` version provided is not compiled with sqlcipher support.
+As a workaround, you can use `this PPA <https://launchpad.net/~linuxgndu/+archive/ubuntu/sqlcipher>`__ to
+install more recent versions of both packages and you can also recompile sqlitebrowser with sqlcipher support following this
```

Comment:
Is this ppa offical? I wouldn't recommend anything from non official PPA since there can be anything in there. Maybe just mention that is available in unofficial PPAs and let people find them?

Reply:
I believe there are no such thing as official PPA. In Ubuntu you have the official repos and then the community PPAs.

Reply:
I have reworded it by adding a warning and removed the link to that PPA.